BrowserKit/Sources/WebEngine/Security/SecurityManager.swift

```diff
@@ -54,30 +47,4 @@ public class DefaultSecurityManager: SecurityManager {
         let urlIsNotComposedOnlyOfAScheme = url.absoluteURL.absoluteString.lowercased() != scheme + ":"
         return isValidScheme && urlIsNotComposedOnlyOfAScheme
     }
-
-    // MARK: - Redirection
```

Comment:
I am cleaning this up for now. I remember what I wanted to achieve with this, but I think now this is trying todo too many things. So until further notice I am removing this commented code.